app/questionnaire/questionnaire_schema.py

```diff
@@ -407,9 +422,9 @@ def _is_list_name_in_rule(self, when_rule: Mapping, list_name: str) -> bool:
         return False
 
     @staticmethod
-    def get_operands(rules: Mapping) -> list:
+    def get_operands(rules: Mapping[str, Any]) -> Sequence[Any]:
         operator = next(iter(rules))
-        operands: list = rules[operator]
+        operands: Sequence[Any] = rules[operator]
```

Comment:
`Any` doesn't have a purpose here, it's the same as you'd give this sequence no attribute.

Reply:
This still need changing.

Reply:
I did miss this one though, thanks, fixed